protos/flow.proto

```diff
@@ -87,6 +87,7 @@ message RenameTablesInput {
   repeated RenameTableOption rename_table_options = 3;
   optional string soft_delete_col_name = 4;
```

Comment:
this was written so we can indicate soft delete being disabled with nil, use that instead of a seperate boolean

we should use optional everywhere else as well, but will break protos so we can do it at v0.18